app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  def new
+    @post = Post.find(params[:post_id])
```

Comment:
[OPTIONAL]

- [ ] This highlighted piece of code is not need in the new action for it to function as it should, please kindly keep your code clean by removing it :+1: :sparkles: